packages/lexical-rich-text/src/index.ts

```diff
@@ -669,6 +675,10 @@ export function registerRichText(
     editor.registerCommand<KeyboardEvent>(
       KEY_BACKSPACE_COMMAND,
       (event) => {
+        // @ts-ignore event.target will be a HTMLElement here
+        if (isTargetWithinDecorator(event.target)) {
```

Comment:
The docs I've seen around this issue appear to suggest just casting as below: 

```suggestion
        if (isTargetWithinDecorator(event.target as HTMLElement)) {
```